OADP-5634: Add new structure for 3scale user story (#89068)
openshift/openshift-docs: modules/restoring-the-3scale-operator-secrets-and-apimanager.adoc

//included in backing-up-and-restoring-3scale-by-using-oadp.adoc assembly

[id="restoring-the-3scale-operator-secrets-and-apimanager_{context}"]
= Restoring the 3scale Operator, secrets, and APIManager

> You can restore the 3scale Operator resources, and `Secret` and APIManager custom resources (CR) by using the following procedure.

Reviewer: ```suggestion
You can restore the Red{nbsp}Hat build of 3scale Operator and `Secret` and APIManager custom resources (CRs) by using the following procedure.
```
why?

.Prerequisites

> * You backed up the 3scale Operator.

Reviewer: ```suggestion
* You backed up the Red{nbsp}Hat build of 3scale Operator.
```
I would fix this naming throughout.

> * You backed up the MySQL and Redis databases.

Reviewer: ```suggestion
* You backed up the MySQL and Redis databases.
```
Again, are these required databases, supported databases, just examples for any customer databases?

* You are restoring the database on the same cluster, where it was backed up. 
+

> If you are restoring on a different cluster, install and configure {oadp-short} with `nodeAgent` enabled on the destination cluster. Ensure that the {oadp-short} configuration is same as it was on the source cluster.

Reviewer: ```suggestion
If you are restoring the Operator to a different cluster that you backed up from, install and configure {oadp-short} with `nodeAgent` enabled on the destination cluster. Ensure that the {oadp-short} configuration is same as it was on the source cluster.
```

  - backuprepositories.velero.io
  itemOperationTimeout: 4h0m0s
----

> <1> Restoring the 3scale Operator's backup

Reviewer: ? this clearly needs some explanatory text

----
$ oc scale deployment threescale-operator-controller-manager-v2 --replicas=0 -n threescale
----
+
.Example output
+
[source,terminal]
----
deployment.apps/threescale-operator-controller-manager-v2 scaled
----

> . Create a YAML file with the following configuration to restore the `Secret`:

Reviewer: ```suggestion
. Create a `restore-secret` YAML file with the following configuration to restore the `Secret` CR:
```

  - backuprepositories.velero.io
  itemOperationTimeout: 4h0m0s
----

> <1> Restoring the `Secret` backup.

Reviewer: explanatory text, e.g., "Use the name you specified in the `metadata.name` field of the backup YAML", etc.

> . Restore the `Secret` by running the following command:

Reviewer: ```suggestion
. Restore the `Secret` CR by running the following command:
```

  - backuprepositories.velero.io
  itemOperationTimeout: 4h0m0s
----

> <1> Restoring the APIManager backup.

Reviewer: explanatory text needed

> <2> The resources that you do not want to restore.

Reviewer: ```suggestion
<2> List any resources that you do not want to restore.
```

> . Restore the APIManager by running the following command:

Reviewer: ```suggestion
. Restore the `APIManager` CR by running the following command:
```
